hl-todo.el

```diff
@@ -5,7 +5,7 @@
 ;; Author: Jonas Bernoulli <jonas@bernoul.li>
 ;; Homepage: https://github.com/tarsius/hl-todo
 ;; Keywords: convenience
-
+;; Version: 0.1
```

Comment:
I knew that you wouldn't like this commit. :)

I will remove it. But I must admit, I don't understand your advice. I think it would be better if your package could be installed using the default package manager. I also don't see why the information is necessarily invalid. All my packages contain such version headers. I bump them in lock step with git commit tags. This way my packages work without problems on ELPA and MELPA.

Reply:
Reverted the commit.